test/cases/platform/.workloadConfig/ingress/contour-component.yaml

```diff
@@ -11,9 +11,7 @@ spec:
     description: Manage contour component
   resources:
   - ingress-ns.yaml
-  - contour-config.yaml
-  - contour-deploy.yaml
-  - contour-svc.yaml
+  - contour-resources
```

Comment:
@lander2k2 I'm assuming the goal of this implementation would be to be able to recursively pull out the resources, through N levels of nested directories?  If so, I think we should nest a directory in the test as well (e.g. contour-resources/nested/contour-config.yaml).

Reply:
I actually think we should *not* do that.  I would prefer not to make any assumptions about what the user intends to include.  I don't imagine users would have elaborate directory structures.  The linked issue doesn't include that requirement and I tend to agree with the simple approach described there.

Reply:
@lander2k2 @scottd018 I actually think supporting a glob, which would allow some one to pull from a dir, or recurse through sub first might be the way to go here, can probably be bolted on after

Reply:
@lander2k2 you should probably sync with @JefeDavis sometime on that requirement.  I would be surprised if his current use case wouldn't require the use of nested directories based upon what I've seen and how his customer has their structure laid out.  I do think the glob implementation may get us there as @JefeDavis described above.

Reply:
The other option is of course to get this merged in, and then add another issue to add in nested directories with globs (if we feel that's what is needed), which feels like maybe the best approach here.

Reply:
OK looks like recursively pulling from subdirs is the consensus.  I'll add it.

Reply:
It looks like filepath glob will return matching files meaning that you wouldn't need to walk the folder structure anymore, this would allow you to control if you want to recurse or not, (i.e. mydir vs mydir/**)